src/mainboard/clevo/mtl-h/devicetree.cb

```diff
@@ -26,61 +26,28 @@ chip soc/intel/meteorlake
 	register "pch_pm_energy_report_enable" = "1"
 
 	# Thermal
-	register "tcc_offset" = "30" # TCC of 100C
+	register "tcc_offset" = "30" # TCC of 80C
 
-	# Power limits
+	# Power limits, why PL1 = 35W if TDP is 28W?
```

Comment:
I bumped it a bit to improve performance, because the cooling system can handle it

Reply:
Then why we hit overheating issues?

Reply:
I don't know yet. Did rolling it back to 28W fix that issue?

Reply:
Didn't test that yet, thus draft. For now looking for things that may be wrong or affect the heat generation

Reply:
Reverted back to 35W and found possible culprit of overheating: https://github.com/Dasharo/ec/pull/62#discussion_r1750048659

Also added PsysPmax setting to coreboot.